### PR TITLE
Allow sidenav contents to scroll Y

### PR DESCRIPTION
**Pull request by yale**

### CODE REVIEW

File: src/components/SideNavBar/style.scss [hunk 1]
```diff
@@ -51,6 +51,7 @@
   box-sizing: border-box;
   border-right: 1px solid $cBorder;
   padding: 20px 0 50px 0;
+  overflow-y: scroll;
```
Comment (maniator) [inline src/components/SideNavBar/style.scss]: weird that it did not do this by default

**Review by maniator — APPROVED** [review]
The lines changed seem off.

But LGTM

**Review by pseuyi — APPROVED** [review]

**Comment by pseuyi** [comment]
removed `/dist`?

**Comment by yale** [comment]
@pseuyi 	@maniator I ran `yarn run pack` and it replaced the CSS with a minified version... not sure why it hasn't always been minified tbh

**Comment by maniator** [comment]
curious as to why we are even committing the dist folder

**Review by kangax — APPROVED** [review]
Kinda bizarre that we have SideNavBar in plasma...

**Pull request merged**